tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1422,9 +1459,14 @@ func visitProductPage(timeout time.Duration, wantStatus int, headers ...*header)
 }
 
 // visitWithApp visits the given url by curl in the given container.
-func visitWithApp(url string, pod string, container string, num int) error {
-	cmd := fmt.Sprintf("kubectl exec %s -n %s -c %s -- bash -c 'for ((i=0; i<%d; i++)); do curl -m 0.1 -i -s %s; done'",
+func visitWithApp(url string, pod string, container string, num int, podLocation bool) error {
+	var cmd string
+	cmd = fmt.Sprintf("kubectl exec %s -n %s -c %s -- bash -c 'for ((i=0; i<%d; i++)); do curl -m 0.1 -i -s %s; done'",
 		pod, tc.Kube.Namespace, container, num, url)
+	if remoteCluster {
+		cmd = fmt.Sprintf("kubectl exec %s -n %s -c %s --kubeconfig=%s -- bash -c 'for ((i=0; i<%d; i++)); do curl -m 0.1 -i -s %s; done'",
+			pod, tc.Kube.Namespace, container, tc.Kube.RemoteKubeConfig, num, url)
+	}
```

Comment:
guess that both local and remote will be 'visited'. There is the Clusters field in the tc.Kube object, which can be iterated to cover both. The Clusters field is intended to replace kubeconfig and remoteKubeConfig.